packages/pocket-ic/src/lib.rs

```diff
@@ -573,6 +573,14 @@ impl PocketIc {
         runtime.block_on(async { self.pocket_ic.advance_time(duration).await })
     }
 
+    /// Get the controllers of a canister.
+    /// Panics if the canister does not exist.
```

Comment:
I feel like I have asked this before on a similar change, but don't remember the answer: Why panic instead of returning a Result?

Reply:
I guess test code may fail fast, and there is no need to recover

Reply:
Other operations such as getting cycles balance also have non-optional return values, i.e., panic in case of canister not found.

Reply:
Regarding reasons: if you're unsure, you can check if the canister exists using a dedicated function; in most cases, I'd assume tests would unwrap anyway.